rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/mget/40_routing.yml

```diff
@@ -8,11 +8,6 @@ routing:
             index:
               number_of_shards: 5
               number_of_routing_shards: 5
-              auto_expand_replicas: 0-1
-
-  - do:
-      cluster.health:
-          wait_for_status: green
```

Comment:
Is it safe to completely remove this section? Should we at least wait for yellow?

Reply:
Seems to be safe. For some of the tests changed here and in my PR linked above, the original test didn't even have the wait, see e.g. https://github.com/elastic/elasticsearch/blob/8.8/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/mget/40_routing.yml#L47-L64.

The get tests use this setup and they seem to be ok. The writes would have to wait I think anyway. See the comment of this commit https://github.com/elastic/elasticsearch/commit/50104020579a36d12899d7a2a66f90c201cf33b3.